payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/PhoneNumberElementUI.kt

```diff
@@ -2,38 +2,37 @@ package com.stripe.android.ui.core.elements
 
 import androidx.annotation.RestrictTo
 import androidx.annotation.StringRes
-import androidx.compose.animation.AnimatedVisibility
-import androidx.compose.animation.ExitTransition
-import androidx.compose.animation.fadeIn
-import androidx.compose.foundation.layout.Box
-import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.text.KeyboardActions
 import androidx.compose.foundation.text.KeyboardOptions
-import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.runtime.collectAsState
 import androidx.compose.runtime.getValue
-import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
-import androidx.compose.runtime.setValue
-import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.focus.FocusDirection
 import androidx.compose.ui.focus.FocusRequester
 import androidx.compose.ui.focus.focusRequester
-import androidx.compose.ui.focus.onFocusChanged
 import androidx.compose.ui.platform.LocalFocusManager
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.input.ImeAction
 import androidx.compose.ui.text.input.KeyboardType
 import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import com.stripe.android.ui.core.R
-import com.stripe.android.ui.core.paymentsColors
+
+@Preview
+@Composable
+fun PhoneNumberElementPreview() {
```

Comment:
Yeah, I don't have screenshot tests for Link yet because it's still changing a lot, so it would slow down development. But the goal is to cover all Link screens closer to GA. Do we have any screenshot tests for just the UI elements?

Reply:
Yeah. It's actually ideal to test smaller elements like this one rather than whole screens. There's a couple of examples of this already. 

https://github.com/stripe/stripe-android/blob/master/paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/screenshot/GooglePayDividerScreenshot.kt